api/config/sanctum.php

```diff
@@ -1,5 +1,13 @@
 <?php
 
+$local = env('APP_ENV') === 'local' ? [
+    env('APP_DOMAIN') . ':8080',
+    'localhost',
```

Comment:
I guess this should also be nawhas.test ?

Reply:
The `env('APP_DOMAIN')` bit takes care of the `nawhas.test` scenario